content/en/docs/rules/custom-ruleset.md

```diff
@@ -0,0 +1,383 @@
+  output: The process %proc.name has used the syscall %evt.type
+```
+
+The previous rule would seldom trigger because it relies on a specific PID (Process ID) for the process, which may rarely occur unless executed within a container. While this example represents an extreme case, it serves to highlight the consequences of being excessively specific when defining a condition. It emphasizes that overly specific conditions can lead to infrequent triggering or potentially no triggering at all in real-world scenarios.
+
+## Tuning a Rule by adding Exceptions to it
+
+Adding exceptions to an existing Falco rule is a useful approach when you want to exclude specific scenarios from triggering that rule. Instead of using the `and not` operators in the condition, which can make the condition more complex and harder to understand, Falco provides a recommended method for handling exceptions.
+
+To add exceptions to a rule, you can utilize the `exceptions` key within the rule definition. By specifying one or more conditions under the `exceptions` key, you can define scenarios where the rule should not be triggered, even if the main condition is satisfied. This approach enhances the readability and maintainability of the rule by explicitly stating the exceptions separately.
+
+Considering the following simplified rule:
+
+```
+- rule: Launch Privileged Container
+  desc: Detect the start of a privileged container.
+  condition: >
+    container_started and container
+    and container.privileged=true
+  output: Privileged container
+```
+
+One way of adding exceptions would be using the `and not` combination explained above:
+
+```
+- list: trusted_images
+  items: [docker.io/user/image1, quay.io/user/image2]
+
+- rule: Launch Privileged Container
+  desc: Detect the start of a privileged container. Exceptions are made for known trusted images.
+  condition: >
+    container_started and container
+    and container.privileged=true
+    and not ( container.image.repository in (trusted_images) or
+              container.image.repository startswith registry.local/ )
+  output: Privileged container
+  priority: debug
+```
+
+The same example using the `exceptions` key:
+
+```
+- rule: Launch Privileged Container
+  desc: Detect the start of a privileged container.
+  condition: >
+    container_started and container
+    and container.privileged=true
+  output: Privileged container
+  priority: debug
+  exceptions:
+    - name: trusted_images
+      fields: [container.image.repository]
+      comps: [in]
+      values:
+        - [(trusted_images)]
+    - name: local_images
+      fields: [container.image.repository]
+      comps: [startswith]
+      values:
+        - [registry.local/]
+```
+
+While this may seem somewhat exaggerated for the provided exceptions, it highlights the true strength of the exceptions key when dealing with a larger number of variables. To demonstrate this, let's explore another example:
```

Comment:
```suggestion
While this may seem exaggerated for the provided exceptions, it highlights the true strength of the exceptions key when dealing with a larger number of variables. To demonstrate this, let's explore another example:
```